src/components/Articles.jsx

```diff
@@ -19,10 +22,46 @@ const Articles = () => {
     })
   }, [topic])
 
+  const handleSortBy = (e) => {
+    let params = { sort_by: e.target.value, order: order }
+    let sort_by = e.target.value
+    setSearchParams(params)
+    setSort(sort_by)
```

Comment:
You could just use params.sort_by with setSort instead of having a separate variable referencing the same thing. Makes for a drier code! :)